test-spark.yaml

```diff
@@ -0,0 +1,121 @@
+package:
+  name: sparkr
+  version: 3.5.3
+  epoch: 1
+  description: Unified engine for large-scale data analytics
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - spark=${{package.full-version}}
+      - spark-3.5=${{package.full-version}}
+      - spark-3.5-scala-2.13=${{package.full-version}}
+    runtime:
+      - openjdk-17
```

Comment:
Is this project supports multiple openjdk versions? If so, we should build for all supported JDKs, example here: https://github.com/wolfi-dev/os/blob/cb8b130b6739d596fab0688fb200a24123743e7f/spark-3.5.yaml#L49-L54